src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -102,14 +119,33 @@ export default {
 		const { loans, totalCount } = await runLoansQuery(
 			this.apollo,
 			{ ...this.flssLoanSearch, ...this.loanSearchState },
-			FLSS_ORIGIN_CATEGORY
+			FLSS_ORIGIN_LENDING_HOME
 		);
 		this.loans = loans;
 		this.totalCount = totalCount;
 		this.backupLoans = this.loans.slice(3);
 	},
+	computed: {
+		filterPageUrl() {
```

Comment:
When first changing the gender dropdown, this filter URL doesn't get updated. And if you select all genders, you get in state where you have a URL like `?gender=`

Reply:
okay this should be fixed now! also moved it to `methods` that way it gets called each time a new CTA is rendered. I think the issue was it wasn't re-rendering since it bases the values off of `flssLoanSearch` which is an object which technically doesn't get updated? And also put in a fallback case for `?gender=` to replace empty strings with `undefined`

Reply:
It's likely related to Vue not detecting due to the property started out undefined (I didn't check if that's what is happening here, but Vue can't detect that situation). But yes, a method works fine.